Datastore/tests/Unit/EntityTest.php

```diff
@@ -66,7 +66,7 @@ public function testCreateEntity()
         $this->assertNull($entity->nonExistentMagicProperty);
         $this->assertObjectNotHasAttribute('nonExistentMagicProperty', $entity);
 
-        $this->assertTrue(isset($entity->magicProperty));
+        $this->assertObjectHasAttribute('magicProperty', $entity);
```

Comment:
We will need to revert this, as the assertion is failing. The `Entity` class implements the magic methods `__set`/`__get` so when you add a property to the class it is actually being stored in an internal data structure.